week-1-practise/level-2/2-format-currency.js

```diff
@@ -6,7 +6,9 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(value) {
+  return "£" + value.toFixed(2);
+}
```

Comment:
In the context of the question asked, your answer is right. However, it's worth bearing in mind that the function is now very firmly tied to being of use with GBP (£) only. Can you think of a way in which you might make the function more generic?